airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -207,13 +235,22 @@ async def airbyte_ci(
     ci_git_user: str,
     ci_github_access_token: str,
     ci_report_bucket_name: str,
+    ci_artifact_bucket_name: str,
     ci_gcs_credentials: str,
     ci_job_key: str,
     s3_build_cache_access_key_id: str,
     s3_build_cache_secret_key: str,
     show_dagger_logs: bool,
 ):  # noqa D103
     ctx.ensure_object(dict)
+
+
+    if enable_dagger_run and not is_current_process_wrapped_by_dagger_run():
```

Comment:
In what case do we start out not wrapped by dagger run, if we only have the one entrypoint now?

Reply:
Actually, I take it back, looks like this is now the entrypoint, so we'll almost always _not_ be wrapped in dagger run and we do the wrapping here. In which case... is this pretty much only to test that someone didn't append `dagger run` themselves?

Reply:
If we're the only ones adding that env var I'm not sure how that works either 🤔